conda/recipes/mantidworkbench/bld.bat

```diff
@@ -0,0 +1,23 @@
+mkdir build && cd build
+
+cmake ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_LIBDIR=%LIBRARY_LIB% ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCONDA_BUILD=True ^
+    -DCONDA_ENV=True ^
+    -DENABLE_PRECOMMIT=OFF ^
+    -DENABLE_DOCS=OFF ^
+    -DMANTID_FRAMEWORK_LIB=SYSTEM ^
+    -DMANTID_QT_LIB=SYSTEM ^
+    -DENABLE_WORKBENCH=ON ^
+    -DCPACK_PACKAGE_SUFFIX="" ^
+    -DWORKBENCH_SITE_PACKAGES=%SP_DIR% ^
+    -DUSE_PRECOMPILED_HEADERS=OFF ^
+    ..
+
+if errorlevel 1 exit 1
+cmake --build . --config Release
+cmake --build . --config Release --target docs-qthelp
+cmake --build . --config Release --target install
+if errorlevel 1 exit 1
```

Comment:
Just curious, why are these windows build files named `bld.bat` rather than `build.bat`?

Reply:
It's the standard naming convention for `conda-build`: https://docs.conda.io/projects/conda-build/en/latest/resources/build-scripts.html

I confess I don't know why `build.bat` is not used. My assumption is that it must clash somewhere with something else called that...